packages/design-system-docs/src/pages/components/Button/button.example.html

```diff
@@ -67,26 +107,30 @@ <h6 class="preview__label">Success button</h6>
 </button>
 
 <h6 class="preview__label">Transparent button</h6>
+<p><code>.ds-c-button .ds-c-button--transparent</code></p>
 <button type="button" class="ds-c-button ds-c-button--transparent">Default</button>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--hover">Hover</button>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--active">
   Active
 </button>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--focus">Focus</button>
 <button type="button" class="ds-c-button ds-c-button--transparent" disabled>Disabled</button>
+<div class="ds-u-margin-bottom--1">
+  <h6 class="preview__label">Size variant buttons</h6>
+  <p><code>.ds-c-button .ds-c-button--small</code></p>
+  <button type="button" class="ds-c-button ds-c-button--small">
+    Small
+  </button>
+  <p><code>.ds-c-button .ds-c-button--big</code></p>
+  <button class="ds-c-button ds-c-button--big">
+    Big
+  </button>
+</div>
```

Comment:
For size variant buttons, I do like the previous version which shows the `small`, `default`, `Big` buttons as that gives a good comparison of the size variant